packages/perseus/src/widgets/interactive-graphs/mafs-graph.tsx

```diff
@@ -77,16 +80,24 @@ export const MafsGraph = (props: MafsGraphProps) => {
                 <View
                     className="mafs-graph"
                     style={{
-                        height,
-                        width: "intrinsic",
                         position: "relative",
                         padding: "25px 25px 0 0",
                         boxSizing: "content-box",
                         marginLeft: "20px",
                         marginBottom: "20px",
                         pointerEvents: props.static ? "none" : "auto",
                         userSelect: "none",
+                        width,
+                        height,
+                    }}
+                    onKeyUp={(event) => {
+                        if (event.key === "Backspace") {
+                            dispatch(actions.global.deleteIntent());
+                            graphRef.current?.focus();
+                        }
                     }}
+                    ref={graphRef}
+                    tabIndex={0}
```

Comment:
Should the graph be focusable with the tab key? If we want it to be programmatically focusable but not tabbable, we could set `tabIndex={-1}`. ([MDN docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex))

Reply:
@mark-fitzgerald and @nishasy thought it should be 0 because we expect it to be tab-able. Nisha has future work that uses that!